code/modules/mob/living/carbon/crit.dm

```diff
@@ -0,0 +1,65 @@
+/mob/living/carbon/handle_crit()
+
+	/////////////////////////////////////////////
+	//// cogwerks - critical health rewrite /////
+	/////////////////////////////////////////////
+	//// goal: make crit a medical emergency ////
+	//// instead of game over black screen time /
+	/////////////////////////////////////////////
+	if(getBrainLoss() >= 100 && health < 0)
+		Weaken(1)
+		losebreath++
+
+	if(getBrainLoss() >= 120) // braindeath, the real killer
```

Comment:
Here's where you'll want to put the -500 health thing with weighted probability away from O2 damage.